mirage/fixtures/external-storage-services.ts

```diff
@@ -0,0 +1,124 @@
+// TODO: needs iconUri and authUri
+export default [
+    {
+        id: 'bitbucket',
+        name: 'Bitbucket',
+        readOnly: true,
+        supportsCopy: false,
+        supportsUserSpecificRootFolder: false,
+        supportsFileVersioning: false,
+        supportsBulkDownload: false,
+        maxConcurrentDownloads: 2,
+        maxUploadMb: 0,
+    },
+    {
+        id: 'box',
+        name: 'Box',
+        readOnly: false,
+        supportsCopy: true,
+        supportsUserSpecificRootFolder: false,
+        supportsFileVersioning: false,
+        supportsBulkDownload: false,
```

Comment:
I thiiink box supports this. Did you check by chance?

Reply:
Seems like this was a bad copy-paste!